### DISCUSSION TITLE
ntfs-3g 无法挂载NTFS硬盘

### DISCUSSION THREAD

**Discussion by SumameLin** [coolsnowwolf/lede#9520] [Q&A / 快速问答]
mount -t ntfs-3g -o rw /dev/sda1 /mnt/sda1
报错，缺失模块
[root@OpenWrt:/root]#mount -t ntfs-3g -o rw /dev/sda1 /mnt/sda1
modprobe: FATAL: Module fuse not found in directory /lib/modules/4.19.245-android
ntfs-3g-mount: fuse device is missing, try 'modprobe fuse' as root
mount: mounting /dev/sda1 on /mnt/sda1 failed: No such device

**Comment by madneal** [comment]
come cross the same issue, is there any solution?
